### PR TITLE
feat(onboard/contributing): documentation - report a bug

### PR DESCRIPTION
**Pull request by tiloyi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

This PR introduce the documentation for the "Report a bug" page.

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: MZC-125

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: src/pages/Contributing/ReportBug/index.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
 ---
 title: 'Report a bug'
 order: 4
----
\ No newline at end of file
+---
+
+If you discover a bug when using a MOZAIC component, please tell us.
```
Comment (gael-boyenval) [inline src/pages/Contributing/ReportBug/index.md]: We should define a little bit what is a bug and talk about it in the context of sketch libraries too. 
Maybe also add a paragraph to tell that if you are not sure, you can ask in the channel

File: src/pages/Contributing/ReportBug/index.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
----
\ No newline at end of file
+---
+
+If you discover a bug when using a MOZAIC component, please tell us.  
+For this, two solutions are available to you:
+
+## Create an Issue
+
+One way to report a bug is to **create an issue** in the [MOZAIC project on Github](https://github.com/adeo/mozaic-design-system/issues) using the dedicated template.
```
Comment (gael-boyenval) [inline src/pages/Contributing/ReportBug/index.md]: maybe a direct link to the right template ?

File: src/pages/Contributing/ReportBug/index.md (hunk 1 above)
Comment (tiloyi) [reply src/pages/Contributing/ReportBug/index.md]: I updated the file. Tell me if it's a little more convenient for you.

**Comment by tiloyi** [comment]
http://featmzc125documentationreportabug.demo.mozaic-design-system-dtrp.appspot.com/Contributing/ReportBug/

**Pull request merged**